sonar-ws/src/main/java/org/sonarqube/ws/client/measure/ComponentWsRequest.java

```diff
@@ -25,29 +25,54 @@
 
 public class ComponentWsRequest {
   private String componentId;
-  private String componentKey;
+  private String component;
   private List<String> metricKeys;
   private List<String> additionalFields;
   private String developerId;
   private String developerKey;
 
+  /**
+   * @deprecated since 6.6, please use {@link #getComponent()} instead
+   */
+  @Deprecated
   @CheckForNull
   public String getComponentId() {
     return componentId;
   }
 
+  /**
+   * @deprecated since 6.6, please use {@link #setComponent(String)} instead
+   */
+  @Deprecated
   public ComponentWsRequest setComponentId(@Nullable String componentId) {
     this.componentId = componentId;
     return this;
   }
 
+  /**
+   * @deprecated since 6.6, please use {@link #getComponent()} instead
+   */
+  @Deprecated
   @CheckForNull
   public String getComponentKey() {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)